apps/angular/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,17 @@
 import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
+import { MyPipePipe } from './my-pipe.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
   selector: 'app-root',
   template: `
     <div *ngFor="let person of persons; let index = index">
-      {{ heavyComputation(person, index) }}
+      {{ person | myPipe: index }}
     </div>
   `,
+  imports: [NgFor, MyPipePipe],
```

Comment:
good, and you can use the new control flow if you want to practise on new angular features

Reply:
hi!
good call, updated it in the latest commit